addons/website_slides/controllers/main.py

```diff
@@ -434,8 +434,7 @@ def _prepare_additional_channel_values(self, values, **kwargs):
     def _get_top3_users(self):
         return request.env['res.users'].sudo().search_read([
             ('karma', '>', 0),
-            ('website_published', '=', True),
-            ('image_1920', '!=', False)], ['id'], limit=3, order='karma desc')
```

Comment:
To double check with SM team if we are ok to take users with placeholder into account.

Reply:
@tde-banana-odoo could you give us an input here?

Reply:
I think original idea was to avoid having ugly broken image. So indeed if we have nice placeholders this condition can be removed.

Reply:
Thank you for the feedback, I'll consider this comment resolved then.